### PR TITLE
Build window wheels on appveyor

### PR DESCRIPTION
**Pull request by fgregg**
This scripts will

* test python-crfsuite on windows
* on tagged commites, build and deploy wheels to pypi

You'll need to enable appveyor for this repo: https://www.appveyor.com/docs

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+environment:
+  password:
+    secure: uOcv5TOm/CgwOmiLQhNX+g==
```
Comment (fgregg) [inline appveyor.yml]: If you want to appveyor to deploy the wheel to pypi, you'll need to create an encrypted variable for "password=the_zope.wheelbuilder_password" and let me know what it is, I'll incorporate it into the script. https://www.appveyor.com/docs/build-configuration#secure-variables

Comment (kmike) [inline appveyor.yml]: How does it work? I.e. to deploy to pypi one needs an username and a password, right?

**Comment by kmike** [comment]
Awesome, thanks @fgregg!

**Comment by kmike** [comment]
@fgregg it seems MSVC 2010 can't build crfsuite since https://github.com/chokkan/crfsuite/pull/44. Do you have an advice? Can we use MSVC 2015 to build Python wheels, or should crfsuite be fixed, or is there some other solution?

**Pull request merged**